src/components/error/Error404.class.ts

```diff
@@ -0,0 +1,7 @@
+export class Error404 {
+  text: string;
+
+  constructor() {
+    this.text = 'Error 404';
```

Comment:
Better if we use a friendly message here, something like: "Página no encontrada".